src/settings.ts

```diff
@@ -39,6 +39,7 @@ export const SPARKLE_ICON = "/sparkle-icon.png";
 export const DEFAULT_MAP_BACKGROUND = "/maps/Sparkle_Field_Background.jpg";
 export const DEFAULT_VENUE_BANNER = "/assets/Sparkle_Banner_Default.jpg";
 export const DEFAULT_VENUE_LOGO = "/assets/Sparkle_SquareLogo_Default.jpg";
+export const NEW_MESSAGE_SOUND = "/sounds/newmessage.m4a";
```

Comment:
We don't want to use 'magic strings' to import this asset. Please use proper webpack imports as per the previous PR:

- https://github.com/sparkletown/sparkle/pull/1180/files#diff-433083e9966b3a808a0b2dce3e47b0b74152c61cb7d5b6a4ad8aad2592dacd81
- https://github.com/sparkletown/sparkle/pull/1180/files#diff-a22d769c39c4171c61bd19fa96454692ce900a294f3507e3c841a23086b17e47